pie-metal/test_all_layers_divergence.py

```
@@ -0,0 +1,230 @@
+"""
+
+import sys
+import os
+from pathlib import Path
+
+workspace_dir = Path('/Users/seung-seoblee/Workspace/pie')
+sys.path.insert(0, str(workspace_dir / 'backend' / 'backend-python'))
+
+os.environ['PIE_METAL_PYTORCH_MODE'] = '1'
+
+import torch
```

Ruff (0.13.3)

Module level import not at top of file

(E402)